apps/src/templates/sectionAssessments/MultipleChoiceAnswerCell.jsx

```diff
@@ -24,10 +24,26 @@ class MultipleChoiceAnswerCell extends Component {
   static propTypes = {
     percentValue: PropTypes.number.isRequired,
     isCorrectAnswer: PropTypes.bool,
+    answerCell: PropTypes.string,
   };
 
   render() {
-    const {percentValue, isCorrectAnswer} = this.props;
+    const {percentValue, isCorrectAnswer, answerCell} = this.props;
+    if (answerCell) {
+      return (
+        <div>
+          <div>
+            {answerCell}
+          </div>
+          <div style={styles.icon}>
+          {isCorrectAnswer &&
```

Comment:
this block should be indented

Reply:
Since the checkmark is shared no matter the type - it'd be nice to not duplicate that element here. We can talk in person about some options for this.